integration-tests/integration/prisonerProfile/Finances.spec.js

```diff
@@ -0,0 +1,218 @@
+const prisonerQuickLookPage = require('../../pages/prisonerProfile/prisonerQuickLookPage')
```

Comment:
Contents of this file should probably be within `prisonerQuickLookSpec.js` and under a `describe('Finances section'...`